test/performance/specs/site-editor.spec.js

```diff
@@ -168,14 +175,8 @@ test.describe( 'Site Editor Performance', () => {
 		// Enter edit mode.
 		await canvas.locator( 'body' ).click();
 
-		// Insert a new paragraph right under the first one.
-		await canvas
-			.getByRole( 'document', { name: 'Block: Post Content' } )
-			.click();
-		await firstParagraph.click();
-		await pageUtils.pressKeys( 'primary+a' );
-		await page.keyboard.press( 'ArrowRight' );
-		await page.keyboard.press( 'Enter' );
```

Comment:
I assume there's a reason these are all gone here, but are we certain they didn't help reset some state which was helpful for the tests?

Reply:
The reason for those changes is to bring consistency in methodology. To make these changes easier to follow, I've made some _before_ and _after_ recordings and put them together in [this comment](https://github.com/WordPress/gutenberg/pull/52022#issuecomment-1651225241). Hope it helps! 

P.S. 
This test actually needed a small tweak that I pushed [here](https://github.com/WordPress/gutenberg/pull/52022/commits/2c305da1b9353890b15149490d0c2c0e165b4037). The change was intended but still not working as expected, so nice catch! 😄